ci/README.md

```diff
@@ -107,6 +110,38 @@ IMAGE_NAME="envoyproxy/envoy-build-${DISTRO}" IMAGE_ID=my_tag ./ci/run_envoy_doc
 
 This build the Ubuntu based `envoyproxy/envoy-build-ubuntu` image, and the final call will run against your local copy of the build image.
 
+# Building a CentOS 7 Binary
+
+Currently, there is no binary image for CentOS 7 available on Docker Hub. However, a build image based on CentOS 7 has been created and pushed again so you can create your own binary for CentOS 7.
+
+Here are some instructions on how to do that locally.
+
+## Pre-Requisites
+
+For a sufficiently fast build, you should have access to a computer or cloud VM with a minimum of 8GB RAM and 4 CPUs/Cores running a later Linux (CentOS 7, Ubuntu 16.04 or 18.04) with Docker CE installed. The build image requres at least Docker 17.03.0-ce. The latest build image has been tested on Docker CE 18.09.0.
+
+Please refer to the Docker documentation on how to install and setup Docker on your specific OS.
+
+## Building
+
+Once you have cloned this repository onto your build machine, you can run the following command to build a binary for CentOS 7:
+
+`IMAGE_NAME=envoyproxy/envoy-build-centos IMAGE_ID="latest" ./ci/run_envoy_docker.sh 'BAZEL_BUILD_EXTRA_OPTIONS=--copt=-DENVOY_IGNORE_GLIBCXX_USE_CXX11_ABI_ERROR=1 ./ci/do_ci.sh bazel.release.server_only'`
+
+This will pull the latest CentOS build image from Docker Hub.
+
+In case you want to build a binary from a specific tag, you will need to get the git commit hash (the full length hash and not just the first 7 characters) and replace the `latest` with this hash string in the `IMAGE_ID` above.
+
+Please note the important 
+
+`BAZEL_BUILD_EXTRA_OPTIONS=--copt=-DENVOY_IGNORE_GLIBCXX_USE_CXX11_ABI_ERROR=1`
```

Comment:
Is this really something that you want to encourage? Did you try building it against `libcxx`? cc @lizan

Reply:
This is necessary because on CentOS 7, I have only gcc 5.3. Unfortunately, gcc 5.4 was never released on CentOS 7 and the next versions I could install are gcc6 or gcc7. I have tried it with gcc7 but the build failed due to other issues. As stated above, I wanted to get this build going again first and then investigate an upgrade of the toolchain to newer versions.